Tasks/SonarQubePostTest/PRCA/PostComments-Server.ps1

```diff
@@ -142,7 +142,13 @@ function GetModifiedFilesInPR
 #
 function PostDiscussionThreads
 {
-    param ([ValidateNotNull()][Microsoft.VisualStudio.Services.CodeReview.Discussion.WebApi.DiscussionThreadCollection]$threads)
+    param ([Microsoft.VisualStudio.Services.CodeReview.Discussion.WebApi.DiscussionThreadCollection]$threads)
```

Comment:
I removed the ValidateNotNull attribute because PS conflates an empty collection to null